<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ListSettingCardOperator</class>
 <widget class="QWidget" name="ListSettingCardOperator">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string/>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="3">
    <widget class="TransparentToolButton" name="pb_movetop"/>
   </item>
   <item row="0" column="1">
    <widget class="TransparentToolButton" name="pb_moveup"/>
   </item>
   <item row="0" column="2">
    <widget class="TransparentToolButton" name="pb_movedown"/>
   </item>
   <item row="0" column="0">
    <widget class="TransparentToolButton" name="pb_add"/>
   </item>
   <item row="0" column="4">
    <widget class="TransparentToolButton" name="pb_delete"/>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>ToolButton</class>
   <extends>QToolButton</extends>
   <header>qfluentwidgets</header>
  </customwidget>
  <customwidget>
   <class>TransparentToolButton</class>
   <extends>ToolButton</extends>
   <header>qfluentwidgets</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
